pkg/auth/authproviders/oidc/backend_impl.go

```diff
@@ -562,12 +562,21 @@ func (p *backendImpl) Validate(context.Context, *tokens.Claims) error {
 // Helpers
 ///////////
 
-// UserInfo is an internal helper struct to unmarshal OIDC token info into.
+// realmAccess is an internal helper struct to represent Keycloak-specific realm_access claim.
+type realmAccess struct {
+	Roles []string `json:"roles"`
+}
+
+// userInfoType is an internal helper struct to unmarshal OIDC token info into.
 type userInfoType struct {
-	Name   string   `json:"name"`
-	EMail  string   `json:"email"`
-	UID    string   `json:"sub"`
-	Groups []string `json:"groups"`
+	Name        string       `json:"name"`
+	EMail       string       `json:"email"`
+	UID         string       `json:"sub"`
+	Groups      []string     `json:"groups"`
+	RealmAccess *realmAccess `json:"realm_access"`
+	// Claim "account_id" is the claim that represents organisation id within sso.redhat.com.
```

Comment:
I'd suggest to add the link to [this page](https://source.redhat.com/groups/public/it-user/it_user_team_wiki/topic_external_sso_enablements#attributes-needed) as this is quite confusing as a name choice.